analysis/combiners.py

```diff
@@ -160,14 +161,27 @@ def _compute_pmf(self) -> poisson_binomial.PMF:
                                       Optional[SumOfRandomVariablesMoments]]
 
 
+def _combine_list(a: List, b: List) -> List:
+    """Combiners 2 lists in 1 and returns it.
+
+    Warning: it changes arguments.
+    """
+    # Extend the larger list (for performance reasons).
```

Comment:
supernit: I would remove the paragraphs since the performance reasons seem to be the key to this whole function